### PR TITLE
add regression test for #19427

### LINKED ISSUE #19427: wrong integral value
sympy reports 0 on an integral that isn't 0.
```python
from sympy import integrate, Symbol, sqrt

x = Symbol("x")

print(integrate((x ** 4) * sqrt(1 - x ** 2), (x, -1, 1)))
print(integrate((-2 * x ** 2) * sqrt(1 - x ** 2), (x, -1, 1)))
print(integrate((1) * sqrt(1 - x ** 2), (x, -1, 1)))

# sum
print(integrate((x ** 4 - 2 * x ** 2 + 1) * sqrt(1 - x ** 2), (x, -1, 1)))
```
```
pi/16
-pi/4
pi/2
0
```
The last line should be the sum of the previous lines, `5/16 pi`.

### PR DESCRIPTION
**Pull request by nschloe**
Fixes #19427.

NO ENTRY

### CODE REVIEW

**Comment by oscarbenjamin** [comment]
The CI failures are unrelated apart from the release notes one that I have just fixed.

Basically when you open the PR the template has this which should not be removed (you have to add the NO ENTRY yourself):
```
<!-- BEGIN RELEASE NOTES -->
NO ENTRY
<!-- END RELEASE NOTES -->
```

**Comment by nschloe** [comment]
Thanks!

File: sympy/integrals/tests/test_integrals.py [hunk 1]
```diff
@@ -2097,3 +2097,16 @@ def test_issue_20782():
...
     assert integrate(f, (x, -oo, 1)) == 1
     assert integrate(-f, (x, -oo, 1)) == -1
+
+
+def test_issue_19427():
+    # <https://github.com/sympy/sympy/issues/19427>
+    x = Symbol("x")
+
+    # Have always been okay:
+    # assert integrate((x ** 4) * sqrt(1 - x ** 2), (x, -1, 1)) == pi / 16
```
Comment (faze-geek) [inline sympy/integrals/tests/test_integrals.py:2107]: It should not be an issue if you add the correct tests also as regression tests, showing new modifications haven't negatively impacted working examples. You can uncomment these lines.

File: sympy/integrals/tests/test_integrals.py [hunk 2]
```diff
@@ -2097,3 +2097,16 @@ def test_issue_20782():
     f = Piecewise((0, x < 0.0), (1, True)) - Piecewise((0, x < 1.0), (1, True))
     assert integrate(f, (x, -oo, 1)) == 1
     assert integrate(-f, (x, -oo, 1)) == -1
+
+
+def test_issue_19427():
+    # <https://github.com/sympy/sympy/issues/19427>
```
Comment (oscarbenjamin) [inline sympy/integrals/tests/test_integrals.py:2103]: This test should be marked as `@slow`.

File: sympy/integrals/tests/test_integrals.py (hunk 1 above)
Comment (nschloe) [reply sympy/integrals/tests/test_integrals.py:2108]: Seems a little weird to me. If "the more the better" is a test strategy, why not also test `assert integrate((x ** 2) * sqrt(1 - x ** 2), (x, -1, 1)) == pi / 8`?

**Comment by oscarbenjamin** [comment]
Set to automerge once CI passes.

The other CI problem should be fixed now (gh-25542).

Thanks!

**Pull request merged**